Apps/WebClient/src/Server/Startup.cs

```diff
@@ -118,6 +119,19 @@ public void ConfigureServices(IServiceCollection services)
             // Configure SPA
             services.AddControllersWithViews();
 
+            services.AddHsts(options =>
+            {
+                options.IncludeSubDomains = true;
+                options.MaxAge = TimeSpan.FromDays(365);
+                options.Preload = true;
+            });
+            // Configure HTTPS redirection
+            services.AddHttpsRedirection(options =>
```

Comment:
This may not work in the deployed environment as currently service proxy hits port 80.  I think we may need to add work to this ticket to validate that or a better solution would be to use native TLS on all of our services which would require the cert installed.

Reply:
Hey Leo, each OpenShift service (app for us) is actually served on port 8080 (I said 80 above) as per the image build https://github.com/bcgov/healthgateway/blob/dev/Apps/WebClient/Dockerfile 

The OpenShift route does TLS termination and then forwards to the service.  So having redirection is likely going to be problematic.  If this resolves the issue then good but I'm thinking it likely will break something.

Reply:
@sslaws I see, it is more complicated than. I think I can just remove the redirection as the 9616 doesn't ask for doing redirection right?

Reply:
@sslaws per my local testing, we need the redirection for this work. Let's discuss the 2 options that you suggested above then (1) add work to this ticket to validate that or (2) a better solution would be to use native TLS on all of our services which would require the cert installed.

Reply:
Per my discussion with @sslaws today, we will deploy this to DEV and I can verify this there.